apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -143,7 +143,11 @@ function wpcom_global_styles_enqueue_block_editor_assets() {
  * @return void
  */
 function wpcom_global_styles_enqueue_assets() {
-	if ( ! wpcom_should_limit_global_styles() ) {
+	if (
+		! wpcom_should_limit_global_styles() ||
+		! wpcom_global_styles_in_use() ||
+		! wpcom_global_styles_current_user_can_edit_wp_global_styles()
```

Comment:
Might be worth checking this first to avoid running the more expensive `wpcom_global_styles_in_use` unnecessarily.